napari_allencell_annotator/controller/annotator_controller.py

```diff
@@ -105,17 +110,16 @@ def set_curr_img(self, curr_img: Optional[Dict[str, str]] = None):
         if curr_img is not None:
             self.curr_img = curr_img
             path: str = curr_img["File Path"]
-            if path not in self.annotation_dict.keys():
-                self.annotation_dict.update(
-                    {path: [curr_img["File Name"], curr_img["FMS"]]}
-                )
+            if len(self.annotation_dict[path]) < 3:
```

Comment:
Would be good to comment this. Its kind of confusing so can you explain the `if else`? Don't know what's going on when `len(self.annotation_dict[path]) < 3`
and why you're selecting `self.annotation_dict[path][2::])` in the else statement.